cibuildwheel/options.py

```diff
@@ -6,10 +6,10 @@
 import dataclasses
 import difflib
 import enum
-import functools
 import shlex
 import textwrap
 from collections.abc import Callable, Generator, Iterable, Iterator, Set
+from functools import cached_property, lru_cache
```

Comment:
Minor nit: I prefer absolute imports for things that are not classes. Especially since these are decorators, they will always be on their own line, so `@functools.cached_property` / `@functools.lru_cache` are fine, IMO.